src/components/content/catalog/services/delete/DeleteResult.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { useQueryClient } from '@tanstack/react-query';
+import { Alert } from 'antd';
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { ApiError, Response, ServiceTemplateDetailVo } from '../../../../../xpanse-api/generated';
+import { catalogPageRoute } from '../../../../utils/constants';
+import { getQueryKey } from '../query/useAvailableServiceTemplatesQuery';
+import { useGetDeleteMutationState } from './DeleteServiceMutation';
+
+export function DeleteResult({
+    id,
+    serviceRegistrationState,
+    category,
+}: {
+    id: string;
+    serviceRegistrationState: ServiceTemplateDetailVo.serviceRegistrationState;
+    category: ServiceTemplateDetailVo.category;
+}): React.JSX.Element | undefined {
+    const deleteRequestState = useGetDeleteMutationState(id);
+    const queryClient = useQueryClient();
+    const navigate = useNavigate();
+
+    const onRemove = () => {
+        void queryClient.refetchQueries({ queryKey: getQueryKey(category) });
+        navigate({
+            pathname: catalogPageRoute,
+            hash: '#' + category,
+        });
+    };
+
+    if (
+        deleteRequestState[0] &&
+        serviceRegistrationState === ServiceTemplateDetailVo.serviceRegistrationState.UNREGISTERED
+    ) {
+        if (deleteRequestState[0].status === 'success') {
+            return (
+                <Alert
+                    message='Service Delete Successfully'
+                    description={'Service Delete successfully.'}
```

Comment:
description can be something else - Service removed from the database completely.